week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,14 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
 console.log(result);
 
 // Answer these questions:
-// - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// - What does this program do? Run it and see, or Google some of the methods used ot understand them. answer: Concatenation
+// - How many parameters does the function take? 2
+// - What is the function name? s
+// - Where is the function called? with what parameters?at 5 with parameter code , your
+// - What does the function return? codeyour
+// - How can this function be improved? lacks "future", a third parameter  :)
```

Comment:
I'd also probably say that `s` isn't  a very good choice of name for a function. It should really describe what the function does.